Music app/app/src/main/java/com/example/musicapp/Musics/MusicFile.java

```diff
@@ -21,38 +24,57 @@ public MusicFile(String album, String title, String duration, String path,
         this.id = id;
     }
 
-    public void setColor(int color)
-    {
+    public Uri getUri() {
+        File file = new File(path);
+        return Uri.fromFile(file);
+    }
+
+    public void setColor(int color) {
         this.color = color;
     }
 
-    public String getId()
-    {
+    public String getId() {
         return id;
     }
 
-    public String getTitle()
-    {
+    public String getTitle() {
         return title;
     }
 
-    public String getPath()
-    {
+    public String getPath() {
         return path;
     }
 
-    public String getAlbum()
-    {
+    public String getAlbum() {
         return album;
     }
 
-    public String getArtist()
-    {
+    public String getArtist() {
         return artist;
     }
 
-    public String getDuration()
-    {
+    public String getDuration() {
         return duration;
     }
+
+    @Override
+    public boolean equals(Object o)
```

Comment:
You add equals and hashcode. Why?

Reply:
I needed it for unit tests to compare two MusicFile class-es